app/experimenter/experiments/tests/api/v5/test_mutations.py

```diff
@@ -893,6 +944,32 @@ def test_launch_experiment_valid_with_preview_status(self):
         result = content["data"]["updateExperiment"]
         self.assertEqual(result["message"], "success")
 
+    def test_launch_experiment_valid_with_dirty_status(self):
+        user_email = "user@example.com"
+        experiment = NimbusExperimentFactory.create(
+            is_rollout=True,
+            publish_status=NimbusExperiment.PublishStatus.DIRTY,
+            status=NimbusExperiment.Status.LIVE,
+        )
+        response = self.query(
+            UPDATE_EXPERIMENT_MUTATION,
+            variables={
+                "input": {
+                    "id": experiment.id,
+                    "status": NimbusExperiment.Status.LIVE.name,
+                    "statusNext": NimbusExperiment.Status.LIVE.name,
+                    "publishStatus": NimbusExperiment.PublishStatus.REVIEW.name,
+                    "changelogMessage": "test changelog message",
+                }
+            },
+            headers={settings.OPENIDC_EMAIL_HEADER: user_email},
+        )
+        self.assertEqual(response.status_code, 200, response.content)
+
+        content = json.loads(response.content)
+        result = content["data"]["updateExperiment"]
+        self.assertEqual(result["message"], "success")
```

Comment:
Generally whenever there's an API call to write something to the db, I'd expect to see an assertion that checks that the data is saved properly.